test/test_pyrsh.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import time
+import unittest
+import tempfile
+import shutil
+import sys
+import MockSSH
+import paramiko
+from pyrsh import main
+from argparse import Namespace
+
+def args( cmd, host='127.0.0.1', user='testadmin', password='x', port=9999 ):
+    arg = Namespace(host=host, user=user, password=password, cmd=cmd, port=port)
+    return arg
+
+class PyrshTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        #
+        # command: pwd
+        #
+        def pwd_command_success(instance):
+            instance.writeln("[OK]")
+        def pwd_command_failure(instance):
+            instance.writeln("MockSSH: Supported usage: pwd")
+        command_pwd = MockSSH.ArgumentValidatingCommand('pwd', [pwd_command_success],
+                                                       [pwd_command_failure], *[])
+        #
+        # command: ls
+        #
+        def ls_command_success(instance):
+            instance.writeln("[OK]")
+        def ls_command_failure(instance):
+            instance.writeln("MockSSH: Supported usage: ls l")
+        command_ls = MockSSH.ArgumentValidatingCommand('ls', [ls_command_success],
+                                               [ls_command_failure], *["l"])
+        #
+        # command: date
+        #
+        def date_command_success(instance):
```

Comment:
@GitSergeyGit the suggestion is to test success and failure scenarios for single command, no need to test other commands for now.